app/views/rentals/invoice.html.erb

```diff
@@ -1,120 +1,104 @@
 <% provide( :page_header, "Rental ##{@rental.id} --- #{@rental.dates}" ) %>
 <div class="container">
-  <div class="row">
-    <div class="col-xs-3">
-      <%= image_tag 'umass_logo.jpg', size: 130 %>
-    </div>
-    <div class="col-xs-6">
-      <p>
-        Parking Services <br>
-        University of Massachusetts Amherst<br>
-        51 Forestry Way Amherst, MA 01003-9262<br>
-        phone: (413) 577-PARK <br>
-        fax: (413) 545-4440<br>
-        web: http://parking.umass.edu<br>
-        email: parking@admin.umass.edu<br>
-      </p>
+  <div class="col-xs-3">
+    <%= image_tag 'umass_logo.jpg', size: 130 %>
+  </div>
+  <div class="col-xs-6">
+    <p>
+    Parking Services <br>
+    University of Massachusetts Amherst<br>
+    51 Forestry Way Amherst, MA 01003-9262<br>
+    phone: (413) 577-PARK <br>
+    fax: (413) 545-4440<br>
+    web: http://parking.umass.edu<br>
+    email: parking@admin.umass.edu<br>
+    </p>
```

Comment:
I see this is repeated. Might be worth extracting into a partial incase any of the information changes. For instance, down the line you may wish to have your URL be `https` 😄

Reply:
Do you mean repeated across the two invoice pages, that will be solved in #359